services/wallet/cases/get.user.wallet.value.js

```diff
@@ -0,0 +1,17 @@
+const getAssetPrice = require('../../../helpers/get.asset.price');
+
+const GetUserWalletValue = (walletRepo) => ({
+
+  getUserWalletValue: async (userId, riskProfile) => {
+    const wallet = await walletRepo.getUserkWalletByRisk(userId, riskProfile);
+    const btcPrice = await getAssetPrice('btc');
+    const ethPrice = await getAssetPrice('eth');
+    const usdcPrice = await getAssetPrice('usdc');
+    const value = btcPrice.price * wallet.dataValues.btcQuantity
+        + +ethPrice.price * wallet.dataValues.ethQuantity
```

Comment:
Is there an extra + here?

Reply:
No se por que el linter lo agrega cuando hago fix, pero funciona jaja